olmo/torch_util.py

```diff
@@ -75,7 +75,7 @@ def move_to_device(o: T, device: torch.device) -> T:
     elif isinstance(o, tuple):
         return tuple((move_to_device(x, device) for x in o))  # type: ignore[return-value]
     else:
-        return o
+        return o.to(device)  # type: ignore
```

Comment:
Why did you do this? What case is this handling? This final `else` is supposed to handle the case of other objects that don't have a `.to()` method.